src/rpc/base_channel.rs

```diff
@@ -27,6 +26,12 @@ impl Debug for WebRTCBaseChannel {
     }
 }
 
+impl Drop for WebRTCBaseChannel {
+    fn drop(&mut self) {
+        log::debug!("Dropping base channel {self:?}");
+    }
```

Comment:
i'm a little confused by this logic - where do we actually drop the channel object? or is the whole point of this PR that it's happening elsewhere so we should only log here?

Reply:
`drop()` is what's called when something leaves scope. So if this calls that means the `base_channel` is definitely dropped. The log just provides verification